packages/ng-primitives/schematics/tabs/generator.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright © 2024 Angular Primitives.
+ * https://github.com/ng-primitives/ng-primitives
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { formatFiles, generateFiles, names, Tree } from '@nx/devkit';
+import * as path from 'path';
+import { TabsGeneratorSchema } from './schema';
+
+const panel = 'panel';
+const tab = 'tab';
+const list = 'list';
+const button = 'button';
+
+const panelStyles = `
+:host { }
+
+:host:not([data-active]) {
+display: none;
+}
+`;
+const buttonStyles = `
+:host { }
+:host[data-focus-visible] {}
+:host[data-active] {}
+:host {}
+`;
+
+export async function tabsGenerator(tree: Tree, options: TabsGeneratorSchema) {
+  const { fileName } = names(options.name);
+
+  generateFiles(tree, path.join(__dirname, `files/${tab}`), options.directory, {
+    ...options,
+    ...names(options.name),
+    selector: `${options.prefix}-${fileName}`,
+    styles: `:host { }`,
+    template: `<ng-content />`,
+  });
+
+  generateFiles(tree, path.join(__dirname, `files/${panel}`), options.directory, {
+    ...options,
+    ...names(`${options.name}-${panel}`),
+    selector: `${options.prefix}-${fileName}-${panel}`,
+    styles: panelStyles,
+    template: `<ng-content />`,
+  });
+
+  generateFiles(tree, path.join(__dirname, `files/${panel}`), options.directory, {
```

Comment:
I think this should use `list` instead:

```suggestion
  generateFiles(tree, path.join(__dirname, `files/${list}`), options.directory, {
```